src/api/app/models/bs_request.rb

```diff
@@ -730,7 +730,7 @@ def change_review_state(new_review_state, opts = {})
 
       # we want to check right now if pre-approved requests can be processed
       if go_new_state == :new && accept_at
-        Delayed::Job.enqueue AcceptRequestsJob.new
+        AcceptRequestsJob.perform_later
```

Comment:
If you can easily test a side effect of AcceptRequestsJob then it would be nice to test that calling BsRequest#change_review_state results in that side effect (since we are using the inline queue adapter for ActiveJob in test env). But if thats not easy then its ok we don't need a test. I'm changing this to a Nitpick.